plugins/transformer/basic/.gitignore

```diff
@@ -0,0 +1 @@
+basic
```

Comment:
I'm not married to `basic`, but I chose a name that clarified that advanced transformations would never be in scope, because as we know it so far we cannot add Datafusion within Go in a newer version. I'm not strongly opposed to `toolkit` but MacGyver would laugh in our face 🤷

Reply:
I think `basic` is okay for now. We can always change it later by releasing it under a different name